unittests/testeventscreate.py

```diff
@@ -23,15 +23,13 @@
     EVENT_ATTRS_ATTACHMENTS,
     EVENT_ATTRS_LOCATION,
     EVENT_ATTRS_LOCATION_IDENTITY,
-    EVENT_ATTRS_SBOM,
```

Comment:
we still want this don't we?

Reply:
no this is only used when testing with uploading using sbom hub